src/cdk/a11y/live-announcer/live-announcer.ts

```diff
@@ -109,6 +120,9 @@ export class LiveAnnouncer implements OnDestroy {
     // TODO: ensure changing the politeness works on all environments we support.
     this._liveElement.setAttribute('aria-live', politeness);
 
+    this._setIdentifiersAttribute('aria-labelledby', labelledBy);
+    this._setIdentifiersAttribute('aria-describedby', describedBy);
```

Comment:
We probably want to clear these attributes inside the `clear` method, otherwise they'll persist between `announce` calls.

Reply:
Added in 2nd commit